inc/saiport.h

```diff
@@ -1741,12 +1741,54 @@ typedef enum _sai_port_stat_t
     /** Port stat in drop reasons range start */
     SAI_PORT_STAT_IN_DROP_REASON_RANGE_BASE = 0x00001000,
 
+    /** Get in port packet drops configured by debug counter API at index 1 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_1_DROPPED_PKTS,
+
+    /** Get in port packet drops configured by debug counter API at index 2 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_2_DROPPED_PKTS,
+
+    /** Get in port packet drops configured by debug counter API at index 3 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_3_DROPPED_PKTS,
+
+    /** Get in port packet drops configured by debug counter API at index 4 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_4_DROPPED_PKTS,
+
+    /** Get in port packet drops configured by debug counter API at index 5 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_5_DROPPED_PKTS,
+
+    /** Get in port packet drops configured by debug counter API at index 6 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_6_DROPPED_PKTS,
+
+    /** Get in port packet drops configured by debug counter API at index 7 */
+    SAI_PORT_STAT_IN_CONFIGURED_DROP_REASONS_7_DROPPED_PKTS,
+
```

Comment:
What exactly do these attributes mean? It would be best if there are some examples -_-？